src/lighteval/utils/tokenizers.py

```diff
@@ -0,0 +1,353 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
```

Comment:
We normally use the model tokenizer when loading tasks - what are the cases where we would need this file and to use specific tokenizers?